react/features/video-quality/middleware.ts

```diff
@@ -24,7 +22,7 @@ MiddlewareRegistry.register(({ dispatch, getState }) => next => action => {
             const { resolution } = getState()['features/base/config'];
 
             if (typeof resolution !== 'undefined') {
-                dispatch(setPreferredVideoQuality(Number.parseInt(resolution, 10)));
+                dispatch(setPreferredVideoQuality(Number.parseInt(`${resolution}`, 10)));
```

Comment:
why did we need the string templating?

Reply:
`resolution` can be either number or string, but `parseInt` requires the first param to be string